src/command.coffee

```diff
@@ -185,7 +186,7 @@ compileScript = (file, input, base = null) ->
     else
       compiled = CoffeeScript.compile t.input, t.options
       t.output = compiled
-      if o.map
+      if o.map and not o['inline-map']
```

Comment:
Since browsers wouldn't support this I would say that throwing when both options are used would be the best bet.

Is this okay?